bootcompiler/src/main/scala/org/mozartoz/bootcompiler/transform/TreeCopier.scala

```diff
@@ -6,6 +6,16 @@ import ast._
 import symtab._
 
 class TreeCopier {
+  def copyOptions[U <: StatOrExpr](orig: Node, copy: U): U = {
```

Comment:
Seems fine then, although it would be cleaner to make this part of the Bind* constructor: Pos+onStack or a Bind* to copy from (this would ensure the information cannot be lost).
By the way, a similar mechanism to copy the position used to be named `copyAttrs`:
https://github.com/mozart/mozart2/blob/master/bootcompiler/src/main/scala/org/mozartoz/bootcompiler/transform/TreeCopier.scala#L12